api/app/scripts/process_wiki_dump.py

```
@@ -0,0 +1,224 @@
+            logger.info(f"Found {len(pages)} pages in the dump")
+
+            # Process each page
+            processed_count = 0
+            for page in pages:
+                entry = self.process_page(page)
+                if entry:
+                    self.context[entry['category']].append(entry)
+                    processed_count += 1
+
+            logger.info(f"Successfully processed {processed_count} pages")
+            logger.info(f"Categories breakdown:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)